examples/neighbors/plot_lmnn_illustration.py

```diff
@@ -0,0 +1,150 @@
+"""
+==========================================
+Large Margin Nearest Neighbor Illustration
+==========================================
+
+Illustration of Large Margin Nearest Neighbor.
```

Comment:
I would say that we should here either mention that the example is solely to generate the figure and point the reader to the relevant part of the documentation, or explain the figure (which might be a duplication of the docs, and hence is probably not preferable)

Reply:
Done.